.github/workflows/sanity_checks.yml

```diff
@@ -21,6 +21,33 @@ jobs:
         run: |
           ./tools/sanity_checks.py
 
+  MSYS2:
+    runs-on: windows-latest
+    defaults:
+      run:
+        shell: msys2 {0}
+    strategy:
+      matrix:
+        sys: [ MINGW64, MINGW32, UCRT64, CLANG32, CLANG64 ]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: msys2/setup-msys2@v2
+        with:
+          msystem: ${{matrix.sys}}
+          install: git base-devel
```

Comment:
base-devel is interesting. It installs some autotools stuff but no cmake. Maybe should get rid of it.